fuel/datasets/base.py

```diff
@@ -28,10 +29,6 @@ class Dataset(object):
         'targets')`` for MNIST (regardless of which data the data stream
         actually requests). Any implementation of a dataset should set this
         attribute on the class (or at least before calling ``super``).
-    default_iteration_scheme : :class:`.IterationScheme`, optional
```

Comment:
No, I meant that if you *would* use the default stream, something as simple as we present in the tutorial (training a simple classifier) doesn't work, showing that it's not a sensible default of an iteration scheme.

I see your point of not wanting to know what kind of dataset you're dealing with if you're just going to use something like `BatchedStream` afterwards. While it sounds like a technicality, I like `get_example_stream` better than `get_default_stream` because it clearly shows what to expect in return: A stream of individual examples. It doesn't pretend to be a "default", just a particular way of iterating.